src/components/Card.js

```diff
@@ -27,10 +49,19 @@ export class Card {
   _setEventListeners() {
     this._templateClone
       .querySelector(".card__button")
-      .addEventListener("click", this._likeCard);
+      .addEventListener("click", (event) => {
+        this.handleLike(this.id, this.isLiked)
+          .then((res) => res.json())
```

Comment:
` .then((res) => res.json())` should be only in `Api` because it's the responsibility of this class. 


**Please, correct it everywhere in the code**